modules/ui/success.js

```diff
@@ -189,7 +189,7 @@ export function uiSuccess(context) {
 
     if (showDonationMessage !== false) {
       // support ask
-      const donationUrl = 'https://supporting.openstreetmap.org/';
+      const donationUrl = 'https://openstreetmap.app.neoncrm.com/forms/ohm';
```

Comment:
I’m also open to keeping the OSMF donation link below the OHM donation link, since we benefit in a small way from the OSMF’s ongoing funding for iD maintenance.